packages/editor/src/package/editor.tsx

```diff
@@ -74,6 +76,7 @@ export function SerloEditor(props: SerloEditorProps) {
       <EditStringsProvider value={editStrings}>
         <EditorVariantContext.Provider value={editorVariant}>
           <LtikContext.Provider value={_ltik}>
+            {!isProductionEnvironment ? renderTestEnvironmentWarning() : null}
```

Comment:
```suggestion
            {isProductionEnvironment ? null : renderTestEnvironmentWarning()}
```
For me this is more readable / less cognitive load (but it might mostly be a preference now since this is how other people told me to do it :)